src/stores/game.js

```diff
@@ -285,6 +287,13 @@ export const useGameStore = defineStore('game', {
     },
     opponentLeft() {
       this.players = this.players.filter((player) => player.pNum === this.myPNum);
+      
+      //set opponent as not ready (bug repair)
```

Comment:
Same here

Reply:
I'm confused. Is this comment still there?  I don't see it

Reply:
Says outdated now, might have been weird timing on GitHub’s part.